src/common/utils/api/api.test.ts

```diff
@@ -13,6 +13,10 @@ describe('apiRequest - getTarget', () => {
       name: 'Toimipiste A',
       address: 'Helsinki',
       description: '',
+      links: [
+        // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
This linter rule is a bit complex, not sure do we need this 🤔

Reply:
Couple options come to my mind on how to resolve this situation:

1. We get the link_type returned to us as linkType
2. We map the link_type to linkType at the edge of our app. So that there's only one place where we need to specify this //eslint-disable-next-line rule
3. We create our own eslint rule, which allows snake_case, but only for variable named link_type: https://eslint.org/docs/developer-guide/working-with-rules

I think the 3rd option feels most cleanest, but might include most work?

Reply:
I examined no.1, and it seems that most of the Helsinki APIs follow snakecase-convention. 
I'll test number 2 because it is the easiest. And if it fails, I'll go with number 3.

Reply:
After discussing and thinking about this more, it seems like we will have more variables that will be returned to us in snake_case format. Im worried there will be pain in any of the options; we would have to map all those variables, or to declare an eslint rule for all of those variables. Or then just allow different naming conventions for our variables. Im not sure which would be the option with the least pain? What do you think?

Reply:
Yes, I'm thinking about that too after implementing one normalize function. I'll go with number 3

Reply:
I added custom rules for naming-conventions